### PR TITLE
Merge from upstream/hanoi in order to release 1.3.1 snap

### PR DESCRIPTION
**Pull request by siggiskulason**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number:

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Other information

### CODE REVIEW

File: snap/README.md [hunk 1]
```diff
@@ -89,3 +78,29 @@ or
 sudo snap install edgex-app-service-configurable edgex-app-service-configurable_mqtt
 ```
 **Note** – you must ensure that any configuration values that might cause conflict between the multiple instances (e.g. port, log file path, …) must be modified before enabling the snap’s service.
+
+### Secret Store Usage
+Some profile configuration.toml files specify configuration which requires Secret Store (aka Vault) support, however this snap doesn't fully support secure secrets without manual intervention (see below). The snap can also be configured to use insecure secrets as can be done via docker-compose by setting the option ```security-secret-store=false```. Ex.
+
+```
+sudo snap set edgex-app-service-configurable security-secret-store=false
```
Comment (tonyespy) [inline snap/README.md:86]: Unfortunately it looks like I documented this incorrectly in the upstream PR, the value should `off`, not `false` to disable usage of the secret store. Please see the associated change to the `security-secret-store-env-var.sh`. FYI I kept off in this case even though the underlying environment variable supports `TRUE/FALSE`. 

Maybe we should modify the code to support either and do a case-less comparison?

**Note** - we need to fix this in the upstream repository as well...

**Review by tonyespy — CHANGES_REQUESTED** [review]
One change (see inline comment) needs to be addressed...

File: snap/README.md (hunk 1 above)
Comment (siggiskulason) [reply snap/README.md:86]: Thanks for raising this issue. As discussed in our standup, we should merge this PR as-is and then you were going to create a new PR with this fix.

**Review by tonyespy — APPROVED** [review]
Based upon discussion re: the one issue surfaced, I'm going to approve this PR, and submit a follow-up PR to address the inaccurate README.

**Pull request merged**